packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -76,10 +77,10 @@ export async function handleBundleToMedicalRecord({
   return newBundle;
 }
 
-function buildBundle(
+export function buildDocRefBundle(
```

Comment:
Since we're renaming this, maybe we can make a few additional adjustments?
- name
   - from: `buildBundle`
   - to: `buildDocRefBundleWithAttachment`
- params
   - `patient` > `patientId` (so its clearer that it won't include the patient in the bundle)
   - `url` > `attachmentUrl` (clearer what the URL is for)
   - `conversionType` > `mimeType` (there's no conversion being done here, so no reason to have conversion on the name)
      - `ConsolidationConversionType` already has `json` on it
      - I wanted to suggest changing the type (`ConsolidationConversionType` ties it to conversion, if we changed it we could move this function to a higher level, more global), but it would likely require more changes, so this is fine for now